packages/app-builder-lib/templates/nsis/uninstaller.nsh

```diff
@@ -38,6 +141,34 @@ Section "un.install"
 
   !insertmacro setLinkVars
 
+  # delete the installed files
```

Comment:
The order of operations changed as stated in the commit message. We try to keep as much state as possible in case we will need to restore the files on failure. The users would be really frustrated to see their shortcuts and file associations go if `uninstall.exe --updated` fails while restoring all app's files.

Reply:
I just wanted ahead and did it. Makes for a much better user experience!